extension/content/personas.css

```diff
@@ -90,6 +95,20 @@
   font-weight: bold !important;
 }
 
+#personas-toolbar-button.seamonkey, #personas-toolbar-button.seamonkey:hover, #personas-toolbar-button.seamonkey:hover:active {
+  list-style-image: url("chrome://personas/content/personas_30x30.png");
+  -moz-image-region: rect(0, 30px, 30px, 0px);  
+}
+
+toolbar[iconsize="small"] #personas-toolbar-button.seamonkey {
+  list-style-image: url("chrome://personas/content/personas_20x20.png");
```

Comment:
> I think it is not necessary.

Yeah I thought so. I still think it is better to add it for both safety and consistency reasons.